targets/core

```diff
@@ -113,7 +113,7 @@ fixkeyboardmode
 install --minimal sudo wget ca-certificates apt-transport-https
 
 # Generate and set default locale
-if [ ! -f '/etc/default/locale' ] && hash locale-gen 2>/dev/null; then
+if ! grep "LANG=" /etc/default/locale && hash locale-gen 2>/dev/null; then
```

Comment:
thanks for the patch!  grep should be used with `-q` to silence its output, and also should be used with `2>/dev/null` to avoid output in the case the file doesn't exist.

Finally, you probably want to make sure the `LANG=` line isn't commented out (e.g., if in the future the skeleton file includes an example).

Something like (untested):

`if ! grep -q '^[^#]*LANG=' /etc/default/locale 2>/dev/null && ...`